app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -96,17 +106,24 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
             startActivity(intent)
         }
 
+
+        if(!hasConnectivity(this)){
+            disableButtons()
+        }
```

Comment:
Same problem as in MainActivity, if user doesn't have time to press on one of the buttons before this is called, he will never get the feedback that his connection is what is stopping him from playing since the buttons will be disabled. (I think the Toast display should come with the button disabling.)

Reply:
So my same question as above. The idea would be to have the toast displayed when loading the page?

Reply:
yes